kinomodel/kinaseModel.py

```diff
@@ -0,0 +1,25 @@
+import simtk.openmm as mm
```

Comment:
Our [software guidelines](https://github.com/choderalab/software-development/blob/638a06d2affad456dbc8f75c4d8b010f8119aca4/PYTHON_CODING.md#keep-clean-and-adhere-to-pep8) suggest we adhere to [PEP8 naming conventions](https://www.python.org/dev/peps/pep-0008/) when possible, so we should use
* `snake_case.py` like naming for files (`kinase_model.py` instead of `kinaseModel.py`)
* `CamelCase` like naming for classes (`Kinase` instead of `kinase`)
* `snake_case` for variables (like `pdb_chainid` instead of `pdbChain`)

Reply:
Thanks John! I will fix the naming!